internal/core/input.rs

```diff
@@ -90,7 +90,7 @@ impl Default for InputEventResult {
 /// can specify how to further process the event.
 /// See [`crate::items::ItemVTable::input_event_filter_before_children`].
 #[repr(C)]
-#[derive(Debug, Copy, Clone, Eq, PartialEq)]
```

Comment:
@ogoffart Any possible side-effects this could have? It is necessary since `f64` doesn't implement the `Eq` trait which is type of `Coord` which is being used for the `pos` field inside a few `MouseEvent` variants.

Reply:
This is fine.
This type is not accessible through our public API anyway.